mon-pix/config/icons.js

```diff
@@ -0,0 +1,28 @@
+module.exports = function() {
+  return {
+    'free-solid-svg-icons': [
+      'times',
+      'sign-out-alt',
+      'image',
+      'redo-alt',
+      'lock',
+      'info-circle',
+      'long-arrow-alt-right',
+      'exclamation-circle',
+      'exclamation-triangle',
+      'times-circle',
+      'check-circle',
+      'bars',
+      'bookmark',
+      'thumbs-up',
+      'arrow-left',
+      'eye',
+      'eye-slash',
+      'power-off'
+    ],
+    'free-regular-svg-icons': [
+      'bookmark',
+      'thumbs-up',
+    ],
```

Comment:
Allez je vous dépanne !
```suggestion
'free-solid-svg-icons': [
      'arrow-left',
      'bars',
      'bookmark',
      'check-circle',
      'exclamation-circle',
      'exclamation-triangle',
      'eye',
      'eye-slash',
      'image',
      'info-circle',
      'lock',
      'long-arrow-alt-right',
      'power-off'
      'redo-alt',
      'sign-out-alt',
      'thumbs-up',
      'times',
      'times-circle',
    ],
    'free-regular-svg-icons': [
      'bookmark',
      'thumbs-up',
    ],
```